testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/groups/AbstractGroupTest.java

```diff
@@ -0,0 +1,40 @@
+package org.keycloak.testsuite.console.groups;
+
+import org.jboss.arquillian.graphene.page.Page;
+import org.junit.Before;
+import org.keycloak.admin.client.resource.GroupResource;
+import org.keycloak.admin.client.resource.GroupsResource;
+import org.keycloak.representations.idm.GroupRepresentation;
+import org.keycloak.testsuite.console.AbstractConsoleTest;
+import org.keycloak.testsuite.console.page.groups.CreateGroup;
+import org.keycloak.testsuite.console.page.groups.Groups;
+
+import static org.keycloak.testsuite.util.URLAssert.assertCurrentUrlEquals;
+import static org.keycloak.testsuite.util.URLAssert.assertCurrentUrlStartsWith;
+
+/**
+ *
+ * @author clementcur
+ */
+public abstract class AbstractGroupTest extends AbstractConsoleTest {
```

Comment:
Even though this is out of scope of this PR, thanks for adding it. It's a shame we didn't have UI tests for groups. :(